src/blog/2022/05/sign-up-for-flowforge-cloud.md

```diff
@@ -0,0 +1,57 @@
+---
+title: FlowForge open for everybody
+subtitle: Sign up and start a new Node-RED project within a minute!
+description: With the waitlist now phased out users can sign up and start using FlowForge right away and start developing on new projects.
+date: 2022-05-25 09:00:00.0
+authors: ["zeger-jan-van-de-weg"]
+---
+
+FlowForge wants to enable everyone to build workflows in Node-RED. Since announcing
+[FlowForge Cloud](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/)
+two months ago we've employed a waitlist to was employed to control the pace the
+new users to our platform with which our team could learn what is needed to
+scale up our platform, and create a good first use experience with FlowForge.
+
+Today, anyone can sign up to FlowForge and start a new Node-RED project in under
+a minute!
```

Comment:
```suggestion
FlowForge wants to enable everyone to build workflows in Node-RED. Since announcing
[FlowForge Cloud](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/)
two months ago we've had a waiting list for users to sign up to. That allowed us
to control the pace we were bringing new users onto the platform, learning what
is needed to scale up our platform and continue to improve our first user experience.

Today we have removed the waiting list. Anyone can sign up to FlowForge and start a
new Node-RED project in under a minute!
```